public/app/core/time_series2.ts

```diff
@@ -115,6 +115,15 @@ export default class TimeSeries {
       currentValue = this.datapoints[i][0];
       currentTime = this.datapoints[i][1];
 
+      // Due to missing values we could have different timeStep all along the series
+      // so we have to find the minimum one (could occur with aggregators such as ZimSum)
+      if (i>0) {
+        var previousTime = this.datapoints[i-1][1];
+        if (!this.stats.timeStep || currentTime - previousTime < this.stats.timeStep) {
+          this.stats.timeStep = currentTime - previousTime;
+        }
+      }
+
```

Comment:
made some slight refactoring of this here, https://github.com/grafana/grafana/commit/71eb0f3278b63579efe033b058d2363a3df76add